build-test-analytics-cloud.xml

```diff
@@ -5,6 +5,16 @@
 
 	<macrodef name="build-analytics-cloud-images">
 		<sequential>
+			<if>
+				<equals arg1="${env.CI_TEST_SUITE}" arg2="analytics-cloud-acceptance" />
+				<then>
+					<loadproperties srcfile="${project.dir}/.gradle/gradle.properties" />
+
+					<echo append="true" file="${analytics.cloud.asah.dir}/gradle.properties">&#10;systemProp.repository.private.password: ${systemProp.repository.private.password}</echo>
+					<echo append="true" file="${analytics.cloud.asah.dir}/gradle.properties">&#10;systemProp.repository.private.username: ${systemProp.repository.private.username}</echo>
```

Comment:
@CharlotteFrancis @pyoo47 
shouldn't these use `=`?
`systemProp.repository.private.password=${systemProp.repository.private.password}`
`systemProp.repository.private.username=${systemProp.repository.private.username}`

Reply:
Yes, you are right.. nice catch